packages/pluggableWidgets/bar-chart-web/src/BarChart.tsx

```diff
@@ -0,0 +1,68 @@
+import classNames from "classnames";
+import { createElement, ReactElement, useMemo } from "react";
+import { ChartWidget, ChartWidgetProps } from "@mendix/shared-charts";
+import { usePlotChartDataSeries } from "@mendix/shared-charts/hooks";
+import { BarChartContainerProps } from "../typings/BarChartProps";
+
+const barChartLayoutOptions: ChartWidgetProps["layoutOptions"] = {
+    xaxis: {
+        zeroline: true,
+        fixedrange: true,
+        gridcolor: "#d7d7d7",
```

Comment:
Later, if user wants to change this color, is it possible to do via css?

Reply:
As I see, this is same approach that we have in mendix/charts repo. So, I think it's better to keep it like this.

Reply:
@Keraito does it mean that people can't configure chart colors?

Reply:
They can't through widget properties directly, but they can through advanced custom configurations.